packages/react/src/components/pagination/Pagination.stories.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+
+import { Pagination } from './Pagination';
+
+export default {
+  component: Pagination,
+  title: 'Components/Pagination',
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: {},
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const Basic = (args) => (
+  <Pagination
+    language="en"
+    onChange={() => null}
+    pageCount={5}
+    pageHref={() => '#'}
+    pageIndex={0}
+    paginationAriaLabel="Pagination"
+  />
+);
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const WithTruncation = (args) => (
+  <>
+    <Pagination
+      language="en"
+      onChange={() => null}
+      pageCount={68}
+      pageHref={() => '#'}
+      pageIndex={7}
+      paginationAriaLabel="Pagination 1"
+      siblingCount={0}
+    />
+    <Pagination
+      language="en"
+      onChange={() => null}
+      pageCount={68}
+      pageHref={() => '#'}
+      pageIndex={7}
+      paginationAriaLabel="Pagination 2"
+    />
+    <Pagination
+      language="en"
+      onChange={() => null}
+      pageCount={68}
+      pageHref={() => '#'}
+      pageIndex={7}
+      paginationAriaLabel="Pagination 3"
+      siblingCount={2}
+    />
+  </>
+);
+
+WithTruncation.storyName = 'With truncation';
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const WithoutPrevAndNextButtons = (args) => {
+  return (
+    <Pagination
+      hideNextButton
+      hidePrevButton
+      language="en"
+      onChange={() => null}
+      pageCount={68}
+      pageHref={() => '#'}
+      pageIndex={7}
+      paginationAriaLabel="Pagination"
+    />
+  );
+};
+
+WithoutPrevAndNextButtons.storyName = 'Without prev and next buttons';
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const InteractableExample = (args) => {
+  const [pageIndex, setPageIndex] = useState<number>(5);
+
+  return (
+    <Pagination
+      language="en"
+      onChange={(event, index) => {
+        event.preventDefault();
+        setPageIndex(index);
+      }}
+      pageCount={25}
+      pageHref={() => '#'}
+      pageIndex={pageIndex}
+      paginationAriaLabel="Pagination"
+      siblingCount={1}
+    />
+  );
+};
+
+InteractableExample.storyName = 'Interactable example';
```

Comment:
Should all the examples be interactable?

Reply:
Good idea, I see no reason why not 👍